tools/clang/test/HLSLFileCheck/hlsl/workgraph/test_EmitDxilFunctionProps_MDVals.hlsl

```diff
@@ -0,0 +1,30 @@
+// RUN: %dxc -fcgl -T lib_6_8 %s | FileCheck -check-prefix=CHECK_FCGL %s
+//
+
+// This test will fail if MDVals in DxilMetadataHelper.cpp gets assigned out of bounds.
+// The test is primarily from called_function_arg_nodeoutput.hlsl
+
+// Verify that the correct parameter, the NodeOutputRecord parameter, gets the cast
```

Comment:
Do we still verify parameters?

Reply:
tools\clang\test\HLSLFileCheck\hlsl\workgraph\called_function_arg_nodeoutput.hlsl does still verify parameters, and is the test file where this new test file comes from. Although this new test doesn't necessarily need to verify that the parameter gets the cast, there is an implication that if the cast is successful, then the buffer didn't get assigned out of bounds. 
So, there might be a better way to write a check for this test, but essentially any test that checks for successful compilation here is sufficient.